### PR TITLE
Add known bug for session on some OS.

### PR DESCRIPTION
**Pull request by h4kuna**

### CODE REVIEW

**Comment by dg** [comment]
Probably it is the same on every Windows releases.

**Comment by h4kuna** [comment]
I checked Windows 7 where the bug is not.

**Comment by dg** [comment]
Are you sure? On Windows 8 it doesn't work too.

File: en/sessions.texy [hunk 1]
```diff
@@ -176,6 +176,12 @@ That's why Nette Framework properly cofigures PHP directives to transfer Session
 
 Function ini_set is used for configuring PHP, but unfortunately its use is prohibited at some web hosting services. If it's your case, try to ask your hosting provider to allow this function for you, or at least to configure his server properly.  .[note]
 
+Known bug
+---------
```
Comment (dg) [inline en/sessions.texy]: What about `known limitations`? Bug should be fixed, limitations are not.

File: cs/sessions.texy [hunk 2]
```diff
@@ -176,6 +176,12 @@ Nette Framework proto správně nakonfiguruje PHP direktivy, aby Session ID pře
 
 Pro konfiguraci PHP se používá funkce ini_set, kterou bohužel některé hostingy zakazují. Pokud je to případ i vašeho hostéra, pokuste se s ním domluvit, aby vám funkci povolil nebo alespoň server nakonfiguroval. .[note]
 
+Známé chyby
+-----------
+V prohlížeči Chrome na OS Ubuntu a Windows XP se nemažou cookie, které mají nastavenou expiraci do uzavření prohlížeče. Následující kód není zcela funkční.
```
Comment (dg) [inline cs/sessions.texy]: `Následující kód není zcela funkční.` -> `Sekce s expirací 0 se tedy po zavření prohlížeče nesmaže.` nebo něco takového.

**Comment by h4kuna** [comment]
Chrome has function in the "System" section, **Continue running background apps when Google Chrome is closed.** 

If is checked cookie still live. But on Ubuntu has no effect cookie live everytime. And this checkbox is default checked. It mean 90% people has checked.

I try on Windows 7, where unchecked this option.

Now czech documentation edit, if is all right, I edit english.

**Comment by dg** [comment]
Now it is great, thanks. Maybe the info about Ubuntu should be removed, because it is actually unnecessary - we do not know what operating systems users are using.

File: cs/sessions.texy [hunk 3]
```diff
@@ -176,6 +176,12 @@ Nette Framework proto správně nakonfiguruje PHP direktivy, aby Session ID pře
 
 Pro konfiguraci PHP se používá funkce ini_set, kterou bohužel některé hostingy zakazují. Pokud je to případ i vašeho hostéra, pokuste se s ním domluvit, aby vám funkci povolil nebo alespoň server nakonfiguroval. .[note]
 
+Známá omezení
+-------------
+V prohlížeči Chrome je v nastavení volba **Po ukončení prohlížeče Google Chrome nechat aplikace na pozadí spuštěné**, která je ve výchozím stavu zapnutá. Způsobuje, že cookie která má nastavenou expiraci do uzavření prohlížeče, se nesmaže. Sekce s expirací 0 se tedy po zavření prohlížeče nesmaže. Na Ubuntu (14.04 64bit) tato volba v Chrome nefunguje, cookie žije i po restartu počítače.
```
Comment (Majkl578) [inline cs/sessions.texy]: Čárka za "že cookie".

**Comment by h4kuna** [comment]
Ok Ubuntu removed. Czech typo and english edited. 

My grandma ( 74 years old ) has Ubuntu :)

**Comment by dg** [comment]
The last thing: the PHP snippet is not needed now, isn't it?

**Comment by h4kuna** [comment]
Like example how simulate limitations?

**Comment by dg** [comment]
It will be enough if they will believe ;-)

**Comment by h4kuna** [comment]
Done

**Comment by dg** [comment]
Thank you

**Pull request merged**